packages/react/src/components/Table/Table.story.jsx

```diff
@@ -1406,7 +1437,7 @@ export const WithRowExpansionAndActions = () => {
       id="table"
       columns={tableColumns.map((c) => ({
         ...c,
-        renderDataFunction,
+        renderDataFunction: c.renderDataFunction || renderDataFunction,
```

Comment:
don't overwrite the renderDataFunction in this story if it already exists (like for the new object column)